doc/api/stream.md

```diff
@@ -1848,7 +1848,7 @@ net.createServer((socket) => {
 
 In addition to new Readable streams switching into flowing mode,
 pre-v0.10 style streams can be wrapped in a Readable class using the
-[`readable.wrap()`][] method.
+[`readable.wrap()`][stream-_wrap] method.
```

Comment:
In the footer of the file, there is a ``[`stream.wrap()`]: #stream_readable_wrap_stream`` line, so yes, I think so